packages/react-components/src/components/ActionMenu/ActionMenu.tsx

```diff
@@ -104,28 +102,67 @@ export const ActionMenu: React.FC<ActionMenuProps> = ({
     setIsVisible(true);
   };
 
-  const handleItemClick = (itemOnClick: () => void) => {
-    itemOnClick();
+  const handleItemClick = (itemOnClick?: () => void) => {
+    itemOnClick && itemOnClick();
 
     if (!keepOpenOnClick) {
       setIsVisible(false);
     }
   };
 
+  const getOptionElement = (option: IActionMenuOption, index: number) => {
```

Comment:
I'd suggest to put this element as a separate component into a `components` folder